README.rst

```diff
@@ -20,33 +20,85 @@ configurations remain synchronized with the state of all databases in the
 cluster. PgQ is used for buffering mutation records until they are collected by
 the relay and forwarded to their final destination.
 
+Usage
+=====
+
+PGShovel is a tool which replicates mutations (``INSERT``, ``UPDATE`` and ``DELETE`` operations) made to a PostgreSQL data out of the database and into an ordered stream of transaction-aware mutation messages.  These messages can then be consumed by 3rd party systems as needed.  Commonly, the mutation stream is consumed by data warehouse operations, cache updating services, search index services or can even be processed into other new streams.
```

Comment:
* Will standardize on `pgshovel`.
* Blame my 3rd grade teacher for the double space.  I'll do my best to try to clean them up and not do it anymore.
* Will fix.
* "system" it is.